schemas/addons.json

```diff
@@ -0,0 +1,129 @@
+{
+  "title": "Addon",
+  "description": "Addon",
```

Comment:
Maybe something more explicit like we have [here](https://github.com/mozilla-services/xml2kinto/pull/25/files#diff-9431e21f44d87c781deb4ab6f5979fc4R3)? `An addon blocklist entry`?